### DISCUSSION TITLE
Just wondering if async connection should have a "aclose" method that calls "close"...

### DISCUSSION THREAD

**Discussion by jalvespinto** [sqlalchemy/sqlalchemy#9697] [Ideas]
With sync I use the code bellow to delete tables:
```
 with contextlib.closing(pg_engine.connect()) as con:
     trans = con.begin()
     for table in reversed(database.Base.metadata.sorted_tables):
         con.execute(table.delete())
     trans.commit()
```
I see that contextlib have a 'aclosing' method, but it calls the 'aclose' method from thing passed to it (asyncconnection in my case). Which async connection does not have.

**Comment by CaselIT** [comment]
Hi

I think could add an alias called `aclose`. I guess it's not there because it never came up before

**Reply by CaselIT** [reply]
created https://github.com/sqlalchemy/sqlalchemy/issues/9698 to track it
